src/utils.ts

```diff
@@ -521,3 +531,16 @@ export function hasValidDleq(proof: Proof, keyset: MintKeys): boolean {
 
 	return true;
 }
+
+export function tokenToRawToken(token: string | Token): Uint8Array {
```

Comment:
This feels like bait :P 
I think we should come up with a proper name for this. We call the the serialised version TokenV4 and the object notation TokenV4Template, so maybe this can be TokenV4Binary?